airflow/models/dag.py

```diff
@@ -441,6 +442,7 @@ def __init__(
         owner_links: dict[str, str] | None = None,
         auto_register: bool = True,
         fail_stop: bool = False,
+        display_name: str = None,
```

Comment:
```suggestion
        display_name: str = "",
```

We can get rid of the `None` case entirely. Same for all other classes.